src/univariate/continuous/inversegaussian.jl

```diff
@@ -160,3 +160,45 @@ function rand(rng::AbstractRNG, d::InverseGaussian)
     u = rand(rng)
     u >= p1 ? μ^2 / x1 : x1
 end
+
+#### Fit model
+
+"""
+Sufficient statistics for `InverseGaussian`, containing the weighted
+sum of observations, the weighted sum of inverse points and sum of weights.
+"""
+struct InverseGaussianStats <: SufficientStats
+    sx::Float64      # (weighted) sum of x
+    sinvx::Float64   # (weighted) sum of 1/x
+    sw::Float64      # sum of sample weight
+
+    InverseGaussianStats(sx::Real, sinvx::Real, sw::Real) = new(sx, sinvx, sw)
```

Comment:
AFAICT this innner constructor is the same as the default one, so better omit it.